libdevcore/OverlayDB.cpp

```diff
@@ -18,7 +18,7 @@
 #include "OverlayDB.h"
 #include "TrieCommon.h"
 #include "db.h"
-#include "easylog.h"
+#include "Log.h"
```

Comment:
This can be removed, too. since `Log.h` has been included in `libdevcore/Common.h`, `TrieCommon.h` includes `libdevcore/Common.h`